.cirrus.yml

```diff
@@ -130,15 +130,15 @@ task:
     CC: clang
   matrix:
     - env: {WIDEMUL:  int64,  RECOVERY: yes, ECDH: yes, SCHNORRSIG: yes, ELLSWIFT: yes}
-    - env: {WIDEMUL:  int64,  RECOVERY: yes, ECDH: yes, SCHNORRSIG: yes, ELLSWIFT: yes, CC: gcc}
+    - env: {WIDEMUL:  int64,  RECOVERY: yes, ECDH: yes, SCHNORRSIG: yes, ELLSWIFT: yes, CC: gcc-13}
     - env: {WIDEMUL: int128_struct, ECMULTGENPRECISION: 2, ECMULTWINDOW: 4}
     - env: {WIDEMUL: int128,                 ECDH: yes, SCHNORRSIG: yes, ELLSWIFT: yes}
     - env: {WIDEMUL: int128,  RECOVERY: yes,            SCHNORRSIG: yes}
-    - env: {WIDEMUL: int128,  RECOVERY: yes, ECDH: yes, SCHNORRSIG: yes, ELLSWIFT: yes, CC: gcc}
+    - env: {WIDEMUL: int128,  RECOVERY: yes, ECDH: yes, SCHNORRSIG: yes, ELLSWIFT: yes, CC: gcc-13}
     - env: {WIDEMUL: int128,  RECOVERY: yes, ECDH: yes, SCHNORRSIG: yes, ELLSWIFT: yes, CPPFLAGS: -DVERIFY}
     - env: {BUILD: distcheck}
   brew_script:
-    - brew install automake libtool gcc
+    - brew install automake libtool gcc@13
```

Comment:
Thanks! Done.

Reply:
> nit: What about keeping "gcc" (without @13) and simply adding `ln -sf /usr/local/bin/gcc-* /usr/bin/gcc`? This is independent of the version, and it will make it easier not to screw up in the future?

```
ln -sf /usr/local/bin/gcc-* /usr/bin/gcc
ln: /usr/bin/gcc: Read-only file system
```

Reply:
Done.